_layouts/eip.html

```diff
@@ -18,14 +18,33 @@
 </svg>
 
 <div class="home">
+  <h1 class="page-heading">
+    {% if page.category == "ERC" %}
+      ERC-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}
+    {% elsif page.category != "ERC" %}
+      EIP-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}
+    {% endif %}
```

Comment:
Might as well clean this up while we're here:

```suggestion
    {% if page.category == "ERC" %}ERC{% else %}EIP{% endif -%}
    -{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}
```

Probably should double check that the `-%}` syntax works. I only know it from jinja2 and django.

Reply:
Why is that necessary? Wouldn't this work just as well, then?
```suggestion
    {% if page.category == "ERC" %}ERC{% else %}EIP{% endif -%}-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}
```